libs/backend/src/cast_vote_records/import.ts

```diff
@@ -146,11 +148,12 @@ async function* castVoteRecordGenerator(
 
     // Only relevant for HMPBs
     let castVoteRecordBallotSheetId: number | undefined;
+    const isHandMarkedPaperBallot = Boolean(castVoteRecord.BallotSheetId);
     if (castVoteRecord.BallotSheetId) {
       const parseBallotSheetIdResult = safeParseNumber(
         castVoteRecord.BallotSheetId
       );
-      if (parseResult.isErr()) {
+      if (parseBallotSheetIdResult.isErr()) {
```

Comment:
🙈 Was referencing the wrong result variable here

Will be adding tests soon to get all the new CVR export code back up to 100% code coverage